content/blog/2022-09-26-localstack-x-gitpod-run-cloud-applications-with-localstack-and-gitpod/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: LocalStack x Gitpod - Run cloud applications with LocalStack and Gitpod
+description: Developing and running cloud applications is simplified with Gitpod and LocalStack ✨ Gitpod’s fully automated, ephemeral workspaces provide a great developer experience. LocalStack accelerates your dev and test feedback loop.
+lead: Developing and running cloud applications is simplified with Gitpod and LocalStack ✨ Gitpod’s fully automated, ephemeral workspaces provide a great developer experience. LocalStack accelerates your dev and test feedback loop.
+date: 2022-09-26T11:31:19+05:30
+lastmod: 2022-09-26T11:31:19+05:30
+contributors: ["Waldemar Hummer", "Nancy Chauhan"]
+tags: ['showcase']
+leadimage: "header.jpeg"
+---
+
+{{< img src="header.jpeg" >}}
+
+Shipping applications to production requires them to be battle-tested in the dev environment. The dev environment should mimic the production environment as closely as possible. This comes with a cost: testing cloud applications is cumbersome as they have many moving parts. For instance, if you build a simple Lambda function in AWS, you might use services such as CloudWatch, S3, IAM, etc.
+
+Imagine configuring your project and provisioning these cloud resources in a test environment every time for developing new features. It’s a tiring and time-consuming cycle 😫 -- making the entire development and test feedback loop slow and inefficient.
+
+Developing and running cloud applications is simplified with Gitpod and LocalStack ✨ Gitpod’s fully automated, ephemeral workspaces provide a great developer experience. When you open a new workspace, all the required tools and packages get installed. You can also parallelly work on features in multiple ephemeral workspaces, which can be shared with your peers.
+LocalStack accelerates your dev and test feedback loop. It provides you with a fully functional local AWS cloud stack that gives immediate feedback on your changes without waiting on cloud resources. This saves you time and resources.
+
+In this blog, we describe how you can leverage Gitpod and LocalStack to set up a cloud development workspace without requiring access to your production cloud - allowing you to get productive within seconds.
+
+- [What is LocalStack](#What-is-LocalStack?)
```

Comment:
nit: some of these anchors are not working (looks like they would need to be lower-cased). Wondering if we should remove these links entirely..?

Reply:
Fixed it now